x-pack/plugins/security_solution/public/timelines/components/graph_overlay/translations.ts

```diff
@@ -6,9 +6,9 @@
 
 import { i18n } from '@kbn/i18n';
 
-export const BACK_TO_EVENTS = i18n.translate(
-  'xpack.securitySolution.timeline.graphOverlay.backToEventsButton',
+export const CLOSE_ANALYZER = i18n.translate(
```

Comment:
Anyone against me just inlining this? I don't see it being used anywhere else...